frontend/src/components/grantaccess/GrantAccess.vue

```diff
@@ -323,302 +323,328 @@ function removeForestClientFromList(index: number) {
 </script>
 
 <template>
+    <PageTitle
+        title="Add user permission"
+        subtitle="Add a new permission to a user. All fields are mandatory unless noted"
+    />
     <VeeForm
         ref="form"
         v-slot="{ errors, meta }"
         :validation-schema="formValidationSchema"
         as="div"
     >
-        <PageTitle
-            title="Grant access to a user"
-            :subtitle="'You are editing in ' + selectedApplicationDisplayText"
-        />
         <div class="page-body">
-            <div class="row">
-                <form id="grantAccessForm" class="form-container">
-                    <div class="row">
-                        <label> Select user's domain </label>
-                        <div class="px-0">
-                            <RadioButton
-                                v-model="formData.domain"
-                                :checked="isIdirDomainSelected()"
-                                inputId="idirSelect"
-                                name="domainRadioOptions"
-                                :value="domainOptions.IDIR"
-                                @change="userDomainChange()"
+            <form id="grantAccessForm" class="form-container">
+                <div class="user-radio-group">
+                    <StepInfo
+                        title="User information"
+                        :subtitle="`Enter the user information to add a new user to ${selectedApplicationDisplayText}`"
+                    ></StepInfo>
+                    <label> Select user's domain </label>
+                    <div class="px-0">
+                        <RadioButton
+                            v-model="formData.domain"
+                            :checked="isIdirDomainSelected()"
+                            inputId="idirSelect"
+                            name="domainRadioOptions"
+                            :value="domainOptions.IDIR"
+                            @change="userDomainChange()"
+                        />
+                        <label class="mx-2" for="idirSelect">IDIR</label>
+                    </div>
+                    <div class="px-0">
+                        <RadioButton
+                            v-model="formData.domain"
+                            inputId="becidSelect"
+                            name="domainRadioOptions"
+                            :value="domainOptions.BCEID"
+                            @change="userDomainChange()"
+                        />
+                        <label class="mx-2" for="becidSelect">BCeID</label>
+                    </div>
+                </div>
+
+                <div class="grid-container-element">
```

Comment:
shall we rename this class as well? maybe call it input-with-verify-field, so it actually can explain what it's for

Reply:
This has been renamed as suggested. Thanks!